xcube_cci/dataaccess.py

```diff
@@ -219,8 +246,10 @@ def _get_open_data_params_schema(dsd: DatasetDescriptor = None) -> JsonObjectSch
                 enum=dsd.data_vars.keys() if dsd and dsd.data_vars else None)),
             time_range=JsonDateSchema.new_range(min_date, max_date)
         )
-        if dsd and (('lat' in dsd.dims and 'lon' in dsd.dims) or
-                    ('latitude' in dsd.dims and 'longitude' in dsd.dims)):
+        if dsd and \
+                dsd.crs == 'WGS84' and \
```

Comment:
`dsd.crs == 'WGS84'` are you sure about this? There are other synonyms for the same projection, e.g. "EPSG:4326". It is saver to use pyproj then query `crs.is_geographic`.

Reply:
It is very safe here because the term 'WGS84' is set in cciodp. It could only be different if there is a crs variable present although the crs is geographic. This is not the case for any of the CCI datasets so far, but I think it could be more elegant to ask for a geographic crs.